src/wallet/walletdb.cpp

```diff
@@ -599,6 +599,16 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 return false;
             }
         }
+        else if (strType == "hdchain")
```

Comment:
Yes, because some of them are repeated several times in the file. But maybe I'm remembering some other patch.